test/rollup_test.spec.ts

```diff
@@ -0,0 +1,49 @@
+declare function require(module: string): any;
```

Comment:
You can do:

```
import * as bundle from 'test_es6_bundling';
```

You'll also need a declaration file to get this to compile. I couldn't figure out how to export one, so I just added:

```
// test/test_es6_bundling.d.ts
declare module "test_es6_bundling" {
  var a: any;
  export = a;
}
```

And added it to the `srcs` of the test.